src/main/include/subsystems/Shooter.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include <rev/CANSparkMax.h>
+#include <ctre/phoenix/motorcontrol/can/TalonSRX.h>
+#include <units/math.h>
+#include <frc/controller/PIDController.h>
+#include <frc/DigitalInput.h>
+
+#include "Constants.h"
+
+class Shooter
+{
+    public:
+
+    Shooter();
+
+    void runShooter(const double motor_output);
+
+    void runIntake(const double motor_output);
+
+    void moveToAngle(const double angle);
+
+    void periodic();
+
+    void init();
+
+    rev::CANSparkMax m_firing_motorL;
+    rev::CANSparkMax m_firing_motorR;
+    rev::CANSparkMax m_arm_motorT;
+    rev::CANSparkMax m_arm_motorB;
+
+    rev::SparkMaxAbsoluteEncoder m_arm_encoderT;
+    rev::SparkMaxAbsoluteEncoder m_arm_encoderB;
+
+    ctre::phoenix::motorcontrol::can::TalonSRX m_intake;
+
+    frc::PIDController m_arm_PID;
```

Comment:
In snake_case/underscore_case, the convention is typically to make everything lowercase, even acronyms:

```suggestion
    frc::PIDController m_arm_pid;
```

Additionally, as the arm motors are CAN SPARK MAXes, instead of an `frc::PIDController`, should we be using a `rev::SparkPIDController` here  and fetching it from the motor as in [this REV Robotics example](https://github.com/REVrobotics/SPARK-MAX-Examples/blob/master/C%2B%2B/Position%20PID%20Control/src/main/cpp/Robot.cpp#L23)?

(_By the way, there may be a perfectly good reason to use the FRC PID controller in lieu of the REV one. If so, please disregard._)